src/RWA007-A/scripts/RWA007Deployment.s.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.12;
+
+import "forge-std/Script.sol";
+import "../contract/Import.sol";
+
+interface Changelog {
+    function getAddress(bytes32 what) external returns (address);
+}
+
+interface JoinFab {
+    function newAuthGemJoin(address owner, bytes32 ilk, address gem) external returns (address);
+}
```

Comment:
Ideally, move these interfaces to the end of the file.

Reply:
done